src/PileupTrack.js

```diff
@@ -343,27 +361,27 @@ varying vec4 vColor;
       }
 
       const fetchedTileKeys = Object.keys(this.fetchedTiles);
-      fetchedTileKeys.forEach((x) => {
+      fetchedTileKeys.forEach(x => {
```

Comment:
Are you using an older version of prettier? Parenthesis around anonymous function arguments are on by default as of prettier v2.0

Reply:
Thanks for noticing. I am using the correct version now.